### ISSUE TITLE
autocompletion of Rcpp C++Objects fails

### ISSUE THREAD

**Issue by kevinushey** [rstudio/rstudio#1654]
From https://support.rstudio.com/hc/en-us/community/posts/115009364048-Rstudio-1-1-383-Issue-with-autocomplete-for-object-methods?page=1#community_comment_115005483728:

```
> library(mxnet)
> data = mx.symbol.Variable("data")
> data$
Error in grep(pattern, complete(x), value = TRUE) : 
  argument "pattern" is missing, with no default
```

We need to call `.DollarNames()` with an explicit empty pattern here: 

https://github.com/rstudio/rstudio/blob/b133724b59447c92df096e49ff1292e59ed2e99d/src/cpp/session/modules/SessionRCompletions.R#L1137

as not all implementing methods provide a default of `""` here.

**Comment by jmcphers** [comment]
Backported to 1.1 here: c25739a15ca49fda68c10f6fd2d25266065cb80b

**Comment by dfalty** [comment]
Verified fixed 1.1.392.
